### PR TITLE
Fixes #14632: Debian/Ubuntu install doc fails if lsb_release is not installed

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/14632

### CODE REVIEW

File: src/reference/modules/installation/pages/_partials/debian_repo.adoc [hunk 1]
```diff
@@ -1,7 +1,13 @@
 Add Rudder's package repository:
 
+[source, Bash]
 ----
 
+# If lsb_release is not installed on your machine, change $(lb_release -cs) by your distribution codename.
+# Ex:
+#   stretch for Debian9
+#   bionic  for Ubuntu18
```
Comment (amousset) [inline src/reference/modules/installation/pages/_partials/debian_repo.adoc]: Ubuntu 18.04

File: src/reference/modules/installation/pages/_partials/debian_repo.adoc [hunk 2]
```diff
@@ -1,7 +1,13 @@
 Add Rudder's package repository:
 
+[source, Bash]
 ----
 
+# If lsb_release is not installed on your machine, change $(lb_release -cs) by your distribution codename.
+# Ex:
+#   stretch for Debian9
```
Comment (amousset) [inline src/reference/modules/installation/pages/_partials/debian_repo.adoc]: Debian 9

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**